docs-src/0.7/src/essentials/ui/attributes.md

```diff
@@ -132,56 +132,56 @@ rsx! {
 }
 ```
 
-Attributes lists awill be merged in the order they appear, so later attributes in the list take precedence over earlier attributes. Attribute spreading becomes very useful when refactoring your UI into a reusable component libraries.
+Attributes lists will be merged in the order they appear, so later attributes in the list take precedence over earlier attributes. Attribute spreading becomes very useful when refactoring your UI into a reusable component libraries.
 
 ## Special Attributes
 
 Most attributes in RSX are rendered verbatim, but there are a few exceptions. In some cases, RSX deviates from traditional HTML to simplify development or work better with the ecosystem tools.
 
-### Style attributes
+### Conditional Attributes
```

Comment:
Moved conditional attributes up above class attributes since we use the same syntax for class attributes